onnxruntime/test/autoep/library/ep_arena.h

```diff
@@ -21,7 +21,10 @@ limitations under the License.
 #include <mutex>
 #include <set>
 
+#define ORT_API_MANUAL_INIT
 #include "onnxruntime_cxx_api.h"
+#undef ORT_API_MANUAL_INIT
```

Comment:
What's the reason for this change? It changes the code to require manual initialization but there's no other change to do the initialization. Should something in example_plugin_ep.cc be calling InitApi with the OrtApiBase* the library receives on load?

Reply:
It fixes an ODR violation due to [example_plugin_ep_util.h already manual init](https://github.com/microsoft/onnxruntime/blob/3a94a670e4502b03991ad5f56ac4169c7398dcf1/onnxruntime/test/autoep/library/example_plugin_ep_utils.h#L9C1-L11C27) but this file not.

With the `#pragma detect_mismatch` that I added, the linker told me that there were object files in the example_plugin_ep shared library that had different definition. So, since this shared library was already using manual init, I made all its compilation units agree.

ORT_API_MANUAL_INIT changes the definition the current tree's  Global<T>::api_ initializer and my proposed DefaultInit. I couldn't come up with a way to implementing this that didn't involve some ODR risk.